src/tokenizer.ts

```diff
@@ -935,14 +935,11 @@ export class Tokenizer extends DiagnosticEmitter {
             this.pos = posBefore;
             return Token.IDENTIFIER;
           } else if (isWhiteSpace(c)) {
-            ++pos;
+            pos += numCodeUnits(c);
```

Comment:
```suggestion
            ++pos;
```
`isWhiteSpace` support only 2 bytes whitespaces

Reply:
Yeah, primarily was wondering whether that could change in theory when Unicode adds some new sort of whitespace in the upper ranges one day. Mostly me being careful. Do you think it's not necessary?

Reply:
I do not think it will appear in the future. But we can leave a comment that if isWhiteSpace will support 4-byte whitespace then you should change to a more generalized option for incrementing